File: cs/lc58.cs
public class Solution {
    public int LengthOfLastWord(string s) {
        int i = s.Length-1;
        int count =0;
        while (s[i] == ' ')
            i-=1;
        while (i>=0 && s[i] !=' ')
        {
            count += 1;
            i-=1;
        }
        return count;

    }
}

File: python/lc58.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        i = len(s)-1
        count =0
        while s[i] == ' ':
            i-=1
        while i>=0 and s[i] !=' ':
            count += 1
            i-=1
        return count
